### PR TITLE
Validate BITARD

### PR DESCRIPTION
**Pull request by BitardSolana**
# Validate [BITARD](https://solscan.io/token/2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/Bitard_Solana/status/1778908559268335852?t=eK-B6oHamn9DLcuy7elJ9A&s=19
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/YOUR_ID_HERE

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [ ] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by BitardSolana** [comment]
I attest that the information above is correct. Please validate! The community is anxiously awaiting the Jupiter listing! 

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by defimad — COMMENTED** [review]
All correct, please validate.

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by BitardSolana — COMMENTED** [review]
I can hereby attest that the provided information are accurate, and the token is valid.
 
Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by TheTardinator** [comment]
I confirm this is all correct and legitimate. Let’s get this Jupiter listing done! Thanks 

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by BEN-Z57** [comment]
I verify this information is correct

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by BitardZ** [comment]
Lets get BITARD on Jupiter🔥🔥🔥

**Comment by drock1111** [comment]
I can confirm this information is correct.

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by ibrak1957** [comment]
We are waiting eagerly for this listing. Appreciate if the team could expedite 🙏🏽🙌🏽

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by Hthorp03 — COMMENTED** [review]
The information above is correct. Please validate. We are awaiting the Jupiter listing!

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by farhaansidd** [comment]
I can hereby attest that the provided information is accurate, and the token is valid all the information is correct .

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1358,4 +1358,5 @@ simbawifhat,SWH,ASsnSwFhGVREnuN6YmZQKietjprv731BbETDWQsreAwj,9,https://ibb.co/JR
 BASC,BASC,DM3Y4R7n1HGhP9AkNT6Ex4w1qQTpgq1TyujrMDX11FMT,9,https://arweave.net/7qhG_tPq_PY6xc9yRamTq-o3tceb3z9iioH9383-rBk,true
 test $SHARK,TSTSHRK2,srkxqEaxjNan8LEWVxBkR9jX59KmSp8Eic57H8sFhzw,6,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f736864772d64726976652e67656e65737973676f2e6e65742f324639746a75315748776f637254486a3269526f4c686856716267514a75736455676b594e4c3834774b66342f746573742d736861726b2d6c6f676f2e706e67,true
 DMME.me,DMME,CG9WyPmf9EUQWGFUsaPAnFKuAi4eNZGGgCh7mrvAFm8p,6,https://cf-ipfs.com/ipfs/QmerLhPZW1bXecBtGxFoFs7wByH3A2TCBkujjrSycu1Sys,true
+BITARD,BITARD,2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB,9,https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa,true
```
Comment (TheTardinator) [inline validated-tokens.csv:1387]: All correct. Please validate! The community is looking forward to Jupiter listing!

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by Daslisa** [comment]
The information provided is correct. Please confirm and verify.

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by BitardZ — COMMENTED** [review]
After I have had a opportunity to review BITARD validation, I can confirm all data is CORRECT.

**Comment by Saintkizito777** [comment]
We are waiting eagerly for this listing. Appreciate if the team could expedite

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by kzuik** [comment]
The below is valid!

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by ajtykho — COMMENTED** [review]
Looks great! Please validate.

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Comment by binitnow** [comment]
This is correct

Mint address: 2ZZaE2gNQSPZLEFiNzBewgQhj3wGpx4oDzuAawh4DHBB
Symbol: BITARD
Token Decimals: 9
Token icon: https://image-cdn.solana.fm/images/?imageUrl=https://ipfs.io/ipfs/QmQvturiF2A4YXf4PjuYX9wBUqQyxsT4X79G1yu2y31Uoa
On-chain metadata matches
No duplication on Jupiter's strict list

**Review by defimad — COMMENTED** [review]
Correct

**Review by Jymfletzfur — APPROVED** [review]

**Review by 9yoi — APPROVED** [review]

**Pull request merged**